Google.PowerShell/BigQuery/BqJob.cs

```diff
@@ -396,16 +441,59 @@ public Apis.Bigquery.v2.Data.Job DoQuery()
         }
 
         /// <summary>
-        /// Copy Job main processing function.
+        /// Copy Job main processing function.  
+        /// *This is written using Apis.BigQuery.v2 becuase Cloud.BigQuery did not 
+        /// support Copy opertations at the time of writing.
         /// </summary>
         public Apis.Bigquery.v2.Data.Job DoCopy()
         {
-            throw new NotImplementedException(
-                "Copy jobs are not implemented yet.  Use the *-BqTabledata cmdlets instead.");
+            if (ShouldProcess($"\n\nCopying table(s) to {Destination.TableId}\n\n"))
+            {
+                try
+                {
+                    var copyjob = new Apis.Bigquery.v2.Data.Job()
+                    {
+                        Configuration = new JobConfiguration()
+                        {
+                            Copy = new JobConfigurationTableCopy()
+                            {
+                                DestinationTable = Destination,
+                                SourceTable = Source,
+                                WriteDisposition = (WriteMode != null) ? WriteMode.ToString() 
+                                    : WriteDisposition.WriteIfEmpty.ToString()
+                            }
+                        }
+                    };
+                    var request = new JobsResource.InsertRequest(Service, copyjob, Project);
```

Comment:
So for some reason, this code is giving me an issue:

**var request = new BqCmdlet.Service.Jobs.Insert(copyjob, Project);**

"The type 'Service' does not exist in the type BqCmdlet"  (which it does)

Taking the "BqCmdlet." off doesn't help either.  Still not found.  Yet the code thats already there works.  I have no idea why, but I'm going to refrain from refactoring here and come back to it when i'm done with the other issues.